pkg/api/routes.go

```diff
@@ -151,9 +172,16 @@ func (rh *RouteHandler) CheckVersionSupport(response http.ResponseWriter, reques
 	// work correctly
 	if rh.c.Config.HTTP.Auth != nil {
 		if rh.c.Config.HTTP.Auth.Bearer != nil {
-			response.Header().Set("WWW-Authenticate", fmt.Sprintf("bearer realm=%s", rh.c.Config.HTTP.Auth.Bearer.Realm))
+			response.Header().Add("WWW-Authenticate", fmt.Sprintf("bearer realm=%s", rh.c.Config.HTTP.Auth.Bearer.Realm))
 		} else {
-			response.Header().Set("WWW-Authenticate", fmt.Sprintf("basic realm=%s", rh.c.Config.HTTP.Realm))
+			response.Header().Add("WWW-Authenticate", fmt.Sprintf("basic realm=%s", rh.c.Config.HTTP.Realm))
+		}
+
+		authenticateHeaders := removeHeadersDuplicates(response.Header().Values("WWW-Authenticate"))
```

Comment:
I thought we didn't need this anymore because we gave up on adding the same header multiple times?